applications/asset_tracker_v2/src/modules/data_module.c

```diff
@@ -478,6 +507,9 @@ static void data_encode(void)
 	case -ENODATA:
 		LOG_DBG("No batch data to encode, ringbuffers are empty");
 		break;
+	case -ENOTSUP:
+		LOG_DBG("Encoding of batch data not supported");
```

Comment:
Isn't this an error that the developer has to take care of?

Reply:
Not really. Not all codec backends support encoding of certain types of data. LwM2M for instance does not support encoding nor sending of batch data, and will return `-ENOTSUP` in those instances.